src/pages/App.tsx

```diff
@@ -89,7 +89,7 @@ export default function App() {
       <Route component={DarkModeQueryParamReader} />
       <AppWrapper>
         <ConstructionBanner>
-          Warning: Revo's smart contracts are pending full audits. Please use with caution!
+          Warning: Revo&apos;s smart contracts are pending full audits. Please use with caution!
```

Comment:
Linter gave an error (not just a warning) about this; went ahead and fixed it here.